test/integration/standard_exit_test.exs

```diff
@@ -0,0 +1,98 @@
+defmodule Integration.StandardExitTest do
+  @moduledoc """
+  Integration tests for the full standard exit flows.
+
+  This is running against no childchain/operator or watcher. Instead it mocks
+  out how we need to extract the data, specifically utxo tracking, to create
+  the correct exit data that the contracts can use to start an standard exit.
+  """
+
+  use ExUnit.Case
+
+  alias ExPlasma.Block
+  alias ExPlasma.Client
+  alias ExPlasma.Transaction
+  alias ExPlasma.Transactions.Deposit
+  alias ExPlasma.Transactions.Payment
+  alias ExPlasma.Utxo
+
+  import ExPlasma.Client.Config,
+    only: [authority_address: 0]
+
+  @moduletag :integration
+
+  setup do
+    # On a fresh contract deploy, we need to invoke the contract to add
+    # the exit queue you want explicitly.
+    Client.add_exit_queue(1, <<0::160>>)
```

Comment:
reading this code does not allow me to know what these two parameters are, consider extracting them so that you can name them
```
payment_type = 1

currency = <<0::160>>
```

Reply:
For method calls like this, would it be preferable to use keyword arguments instead?

Reply:
not necessarily, of it's part of a function or flow where both arguments are derived from something and it gives them meaning it's sufficient.
in this case making them module params would suffice to understand what you're doing
```
@payment_type ExPlasma.Transaction.payment()
@payment_currency ExPlasma.Transaction.ethereum_currency()
```
^^made up